catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/TreUtility.java

```diff
@@ -47,6 +47,24 @@ public static Serializable getTreValue(TreGroup tre, String key) {
         return null;
     }
 
+    public static String parseString(Tre tre, String fieldName) {
```

Comment:
This doesn't really do any parsing, so I probably would have called it 'convertToString' or something similar.